contracts/chain-adapters/l2/Ovm_WithdrawalHelper.sol

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: BUSL-1.1
 
-pragma solidity ^0.8.0;
+pragma solidity ^0.8.19;
```

Comment:
If we're going to change Arbitrum to 0.8.0 let's keep this, right?

Reply:
Yeah, we should. It's nice to keep things consistent. https://github.com/across-protocol/contracts/pull/728/commits/b6cae1c347123855208db93f43a03cb0736ba15b